lib/notification_service/notification_service/cli/configuration.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Maybe we should put this file and the `server_config.py` to a submodel under notification_server, such as `notification_server.config`.